skycoin/skywire
# Show the IP in the UI

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #719

 Changes:	
- The visor IP is now used as the default label for visors without label.
- The visor details page now shows the visor IP.
- Now the apps list shows a link for opening the UI of the Skychat app in a new tab. For building the URL, the code uses the visor IP and tries to get the port from the app arguments, but if the port is not there, `8001` is used as the default port.

How to test this PR:
To see the default label, just open the visor list, the IP should be shown as the label of all visors without a label set (previously, the default label was just a part of the public key). For checking the IP of a visor, open the visor details page, the IP should be shown in the right bar. For opening the Skywallet UI, open the app list of a visor and press the link button at the right of the Skychat row.
